modules/bitgo/test/v2/unit/tss/ecdsa.ts

```diff
@@ -344,28 +344,32 @@ describe('Ecdsa tss helper functions tests', function () {
       ];
 
       for (let index = 0; index < config.length; index++) {
         describe(`sendSignatureShare: ${config[index].shareToSend}`, async function () {
           it(`should succeed to send ${config[index].shareToSend}`, async function () {
             const mockSendReq = { from: 'user', to: 'bitgo', share: config[index].mockShareToSendString } as SignatureShareRecord;
-            const response = { from: 'bitgo', to: 'user', share: config[index].mockShareAsResponseString } as SignatureShareRecord;
-            await nockSendSignatureShare({ walletId: wallet.id(), txRequestId: txRequest.txRequestId, signatureShare: mockSendReq, response });
-            const responseAShare = await ECDSAMethods.sendShareToBitgo(bitgo, wallet.id(), txRequest.txRequestId, config[index].sendType, config[index].mockShareToSend, config[index].optionalDShare);
+            const shareRecord = { from: 'bitgo', to: 'user', share: config[index].mockShareAsResponseString } as SignatureShareRecord;
+            await nockSendSignatureShare({ walletId: wallet.id(), txRequestId: txRequest.txRequestId, signatureShare: mockSendReq, response: shareRecord });
+            txRequest.signatureShares = [shareRecord];
+            const response = { txRequests: [{ ...txRequest }] };
+            await nockGetTxRequest({ walletId: wallet.id(), txRequestId: txRequest.txRequestId, response: response });
+            const responseAShare = await ECDSAMethods.sendShareToBitgo(bitgo, wallet.id(), txRequest.txRequestId, config[index].sendType, config[index].mockShareToSend);
             responseAShare.should.deepEqual(config[index].mockShareAsResponse);
           });
 
           it(`should fail if we get an invalid ${config[index].shareReceived} as response`, async function () {
             const mockSendReq = { from: 'user', to: 'bitgo', share: config[index].mockShareToSendString } as SignatureShareRecord;
             const invalidSignatureShare = { from: 'bitgo', to: 'user', share: JSON.stringify(config[index].incorrectReceivedShareString) } as SignatureShareRecord;
             const nock = await nockSendSignatureShare({ walletId: wallet.id(), txRequestId: txRequest.txRequestId, signatureShare: mockSendReq, response: invalidSignatureShare }, 200);
-            await ECDSAMethods.sendShareToBitgo(bitgo, wallet.id(), txRequest.txRequestId, config[index].sendType, config[index].mockShareToSend, config[index].optionalDShare).should.be.rejectedWith(/Excepted share length to be greater than or equal .*/g);
+            txRequest.signatureShares = [invalidSignatureShare];
+            const response = { txRequests: [{ ...txRequest }] };
```

Comment:
NIT: we could reuse that helper function you defined in `modules/bitgo/test/v2/unit/internal/tssUtils/ecdsa.ts` , probably need to add more parameters

Reply:
they are in different files. I don't think it's worth reusing that method